exporters/ostream/include/opentelemetry/exporters/ostream/span_exporter.h

```diff
@@ -110,16 +111,16 @@ class OStreamSpanExporter final : public sdktrace::SpanExporter
 
 #endif
 
-  void print_value(sdktrace::SpanDataAttributeValue &value)
+  void print_value(sdkcommon::OwnedAttributeValue &value)
```

Comment:
There's something wrong about this name. I liked the previous name better. If we want to explain the semantics of some class, it has to be written in the class docs, e.g. `Owned` is not a good prefix for a common class.

Reply:
Instead, what if we simply named it `AttributeValue`, which follows the API's version of `AttributeValue` in [attribute_values.h](https://github.com/open-telemetry/opentelemetry-cpp/blob/master/api/include/opentelemetry/common/attribute_value.h)? It could be referenced with diferent namespaces: `sdk::common::AttributeValue` and `opentelemetry::common::AttributeValue` respectively, like what's done for logs/traces/metrics.

Reply:
@maxgolov  @pyohannes to comment, as I understand johannnes liked the idea of naming it as `OwnedAttributeValue `.

Reply:
I have no strong preference on this. I like that it's named different than the API variety, but it can be something different than `Owned`. I find it slightly more confusing to have it distinguished just by the namespace name, but I don't have a strong opinion on that either. I think it's not very important, as it's mostly an SDK internal concept (most exporters using their own Recordables will most likely not use it).